boards/arm/mimxrt1170_evk/Kconfig.defconfig

```diff
@@ -33,6 +33,19 @@ endchoice
 
 endif #FLASH
 
+config I2C
```

Comment:
remove `config I2C` bits, those should get dealt with by the Kconfig of the driver itself.

Reply:
done